src/components/layout/CardTitle.tsx

```diff
@@ -4,7 +4,15 @@ import type { JSX } from 'preact';
 import type { PresentationalProps } from '../../types';
 import { downcastRef } from '../../util/typing';
 
+type ComponentProps = {
+  /** Wrap the title with this HTML heading tag  */
+  tagName?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5';
```

Comment:
I realized that we lost something in the most recent round of `Card` and `Panel` implementations: we aren't using heading elements. For HTML semantics and accessibility it is best if we do. This prop allows a consumer to use a different heading level than default (h1) if so desired.